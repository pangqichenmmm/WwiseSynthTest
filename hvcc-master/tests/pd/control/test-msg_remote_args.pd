#N canvas 0 44 298 220 10;
#X obj 37 67 loadbang;
#X obj 37 130 print;
#X msg 37 89 hi hello \; param 1;
#X msg 124 95 message 1 \, message 2 \;;
#X connect 0 0 2 0;
#X connect 0 0 3 0;
#X connect 2 0 1 0;
#X connect 3 0 1 0;
