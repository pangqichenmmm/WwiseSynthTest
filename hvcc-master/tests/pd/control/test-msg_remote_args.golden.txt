[@ 0.000] print: hi hello
[@ 0.000] print: message 1
[@ 0.000] print: message 2
